sections/image-banner.liquid

```diff
@@ -24,17 +24,22 @@
 {%- endif -%}
 
 {%- style -%}
-  #Banner-{{ section.id }}::after {
+  #Banner-{{ section.id }}.scroll-animation__trigger::after {
+    opacity: 0;
+  }
```

Comment:
Not a blocking note, but I think this rule could technically be a more generalized one that isn't tied to this specific ID. On the other hand, for code organization's sake, it's kind of nice to keep it here next to the rule below.